app/assets/stylesheets/components/_header.scss

```diff
@@ -1,14 +1,16 @@
 .global-nav {
+  position: relative;
   background-color: $c-blue;
-  box-shadow: 2px 2px 4px rgba(black, 0.8);
+  box-shadow: 0px 2px 22px rgba(black, 0.5);
   padding: 20px;
+  z-index: 3;
   a {
-  color: white;
-  font-family: $f-logo;
-  margin-left: 4%;
-  text-decoration: none;
-  outline-color: orange;
-  @include fluid-size('font-size', 16px, 27px);
+    color: white;
+    font-family: $f-logo;
+    margin-left: 4%;
+    text-decoration: none;
+    outline-color: orange;
```

Comment:
You could have a `border-bottom: 1px solid transparent` and then when hovered/active `border-bottom: 1px solid orange`. I assume the outline is for the underline?

I've never used outline other than troubleshooting. Interested in knowing what @mzlock thinks.